### ISSUE TITLE
Issue with accuracy and loss during training

### ISSUE THREAD

**Issue by anishk12345** [chineseocr/table-detect#17]
Hello,

I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?

**Comment by yangyuqing15715165798** [comment]
> Hello,
> 
> I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?

I also encountered this problem, how many samples do you have for your training set?

**Comment by anishk12345** [comment]
> > Hello,
> > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> 
> I also encountered this problem, how many samples do you have for your training set?

I used around around 500 images on my training set. How much did you use ?

**Comment by yangyuqing15715165798** [comment]
> > > Hello,
> > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > 
> > I also encountered this problem, how many samples do you have for your training set?
> 
> I used around around 500 images on my training set. How much did you use ? 

around 10 images,Do you have GPU training?

**Comment by anishk12345** [comment]
> > > > Hello,
> > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > 
> > > 
> > > I also encountered this problem, how many samples do you have for your training set?
> > 
> > 
> > I used around around 500 images on my training set. How much did you use ?
> 
> around 10 images,Do you have GPU training?

Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me

**Comment by yangyuqing15715165798** [comment]
> > > > > Hello,
> > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > 
> > > > 
> > > > I also encountered this problem, how many samples do you have for your training set?
> > > 
> > > 
> > > I used around around 500 images on my training set. How much did you use ?
> > 
> > around 10 images,Do you have GPU training?
> 
> Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me

I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?

**Comment by anishk12345** [comment]
> > > > > > Hello,
> > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > 
> > > > > 
> > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > 
> > > > 
> > > > I used around around 500 images on my training set. How much did you use ?
> > > 
> > > 
> > > around 10 images,Do you have GPU training?
> > 
> > 
> > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> 
> I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?

Yes, what is the issue you are facing ?

**Comment by yangyuqing15715165798** [comment]
> > > > > > > Hello,
> > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > 
> > > > > > 
> > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > 
> > > > > 
> > > > > I used around around 500 images on my training set. How much did you use ?
> > > > 
> > > > 
> > > > around 10 images,Do you have GPU training?
> > > 
> > > 
> > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > 
> > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> 
> Yes, what is the issue you are facing ?

First of all, if you use GPU training, do you need to change some code in the train.py?

**Comment by anishk12345** [comment]
> > > > > > > > Hello,
> > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > 
> > > > > > > 
> > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > 
> > > > > > 
> > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > 
> > > > > 
> > > > > around 10 images,Do you have GPU training?
> > > > 
> > > > 
> > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > 
> > > 
> > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > 
> > 
> > Yes, what is the issue you are facing ?
> 
> First of all, if you use GPU training, do you need to change some code in the train.py?

Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work.

**Comment by yangyuqing15715165798** [comment]
> > > > > > > > > Hello,
> > > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > > 
> > > > > > > 
> > > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > > 
> > > > > > 
> > > > > > around 10 images,Do you have GPU training?
> > > > > 
> > > > > 
> > > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > > 
> > > > 
> > > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > > 
> > > 
> > > Yes, what is the issue you are facing ?
> > 
> > First of all, if you use GPU training, do you need to change some code in the train.py?
> 
> Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work. 

Yes, I  feel CUDA environment is not configured well.
`Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda-11.0Pb64:/usr/local/cuda-11.0Pb64
2021-08-13 14:44:37.596900: I tensorflow×ream_executorÔatform/default/dso_`

**Comment by yangyuqing15715165798** [comment]
> > > > > > > > > > Hello,
> > > > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > > > 
> > > > > > > 
> > > > > > > around 10 images,Do you have GPU training?
> > > > > > 
> > > > > > 
> > > > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > > > 
> > > > > 
> > > > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > > > 
> > > > 
> > > > Yes, what is the issue you are facing ?
> > > 
> > > First of all, if you use GPU training, do you need to change some code in the train.py?
> > 
> > Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work. 
> 
> Yes, I  feel CUDA environment is not configured well.
> `Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda-11.0Pb64:/usr/local/cuda-11.0Pb64
> 2021-08-13 14:44:37.596900: I tensorflow×ream_executorÔatform/default/dso_`

Can you give me your dataset? Maybe I'll give it a try.

**Comment by yangyuqing15715165798** [comment]
> Hello,
> 
> I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?

Is the accuracy rate.improve now?

**Comment by yangyuqing15715165798** [comment]
> > > > > > > > > > > Hello,
> > > > > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > around 10 images,Do you have GPU training?
> > > > > > > 
> > > > > > > 
> > > > > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > > > > 
> > > > > > 
> > > > > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > > > > 
> > > > > 
> > > > > Yes, what is the issue you are facing ?
> > > > 
> > > > First of all, if you use GPU training, do you need to change some code in the train.py?
> > > 
> > > Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work. 
> > 
> > Yes, I  feel CUDA environment is not configured well.
> > `Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda-11.0Pb64:/usr/local/cuda-11.0Pb64
> > 2021-08-13 14:44:37.596900: I tensorflow×ream_executorÔatform/default/dso_`
> 
> Can you give me your dataset? Maybe I'll give it a try.

I seemed to have found a solution to improve the accuracy,Could you share your dataset? i can try the effect for you

**Comment by anishk12345** [comment]
> > > > > > > > > > > > Hello,
> > > > > > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > around 10 images,Do you have GPU training?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > > > > > 
> > > > > > > 
> > > > > > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > > > > > 
> > > > > > 
> > > > > > Yes, what is the issue you are facing ?
> > > > > 
> > > > > 
> > > > > First of all, if you use GPU training, do you need to change some code in the train.py?
> > > > 
> > > > 
> > > > Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work.
> > > 
> > > 
> > > Yes, I  feel CUDA environment is not configured well.
> > > `Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda-11.0�Pb64:/usr/local/cuda-11.0�Pb64 2021-08-13 14:44:37.596900: I tensorflow�×ream_executor�Ôatform/default/dso_`
> > 
> > 
> > Can you give me your dataset? Maybe I'll give it a try.
> 
> I seemed to have found a solution to improve the accuracy,Could you share your dataset? i can try the effect for you

I used the ICDAR 2013 dataset, https://www.tamirhassan.com/html/dataset.html

**Comment by yangyuqing15715165798** [comment]
> > > > > > > > > > > > > Hello,
> > > > > > > > > > > > > I tried training a model with my own data after annotating it using label-me. But while training I observed that my training loss is not converging nor is my accuracy increasing. It is stuck within a range of values (0.45 - 0.55 for accuracy). Any Idea why this is happening ?
> > > > > > > > > > > > 
> > > > > > > > > > > > 
> > > > > > > > > > > > I also encountered this problem, how many samples do you have for your training set?
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > I used around around 500 images on my training set. How much did you use ?
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > around 10 images,Do you have GPU training?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Yes, I tried training it on a GPU. I tried making some changes to some of the parameters as well. Nothing worked for me
> > > > > > > > 
> > > > > > > > 
> > > > > > > > I'm trying to run with GPU now, but I have some problems, can you teach me how to run with GPU?
> > > > > > > 
> > > > > > > 
> > > > > > > Yes, what is the issue you are facing ?
> > > > > > 
> > > > > > 
> > > > > > First of all, if you use GPU training, do you need to change some code in the train.py?
> > > > > 
> > > > > 
> > > > > Not Needed, just make sure CUDA is set up properly and that tensorflow is detecting your GPU. After that just make sure the model weights file is detected and it will work.
> > > > 
> > > > 
> > > > Yes, I  feel CUDA environment is not configured well.
> > > > `Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/cuda-11.0�Pb64:/usr/local/cuda-11.0�Pb64 2021-08-13 14:44:37.596900: I tensorflow�×ream_executor�Ôatform/default/dso_`
> > > 
> > > 
> > > Can you give me your dataset? Maybe I'll give it a try.
> > 
> > I seemed to have found a solution to improve the accuracy,Could you share your dataset? i can try the effect for you
> 
> I used the ICDAR 2013 dataset, https://www.tamirhassan.com/html/dataset.html

This code needs json format, how to convert ICDAR2013 files into json format? How is your method done, can you give me a copy of the converted data or provide some help？
